config/ignition_citadel_gazebo11_ubuntu_focal.yaml

```
@@ -1,106 +1,93 @@
-name: gazebo11_ubuntu_focal.yaml
+name: ignition_focal_source
```

Not sure if changing the name impacts on your workflow @nuclearsandwich . I'm trying to be accurate, if possible, if not, feel free to drop the suggestion.

```suggestion
name: ignition_citadel_gazebo11_focal_source
```

---

While there's no hard requirement. The name should definitely match the filename. Good catch.